File: csharp/UnitTest/Member/Identification/ClientTest.cs
using NUnit.Framework;
using Alipay.EasySDK.Factory;
using Alipay.EasySDK.Member.Identification.Models;
using System;

namespace UnitTest.Member.Identification
{
    public class ClientTest
    {
        [SetUp]
        public void SetUp()
        {
            Factory.SetOptions(TestAccount.Mini.CONFIG);
        }

        [Test]
        public void TestInit()
        {
            IdentityParam identityParam = new IdentityParam()
            {
                IdentityType = "CERT_INFO",
                CertType = "IDENTITY_CARD",
                CertName = "张三",
                CertNo = "513901198008089876"
            };

            MerchantConfig merchantConfig = new MerchantConfig()
            {
                ReturnUrl = "www.taobao.com"
            };


            AlipayUserCertifyOpenInitializeResponse response = Factory.Member.Identification().Init(
                Guid.NewGuid().ToString(), "FACE", identityParam, merchantConfig);

            Assert.AreEqual(response.Code, "10000");
            Assert.AreEqual(response.Msg, "Success");
            Assert.IsNull(response.SubCode);
            Assert.IsNull(response.SubMsg);
            Assert.NotNull(response.HttpBody);
            Assert.NotNull(response.CertifyId);
        }

        [Test]
        public void TestCertify()
        {
            AlipayUserCertifyOpenCertifyResponse response = Factory.Member.Identification().Certify("bbdb57e87211279e2c22de5846d85161");

            Assert.IsTrue(response.Body.Contains("https://openapi.alipay.com/gateway.do?alipay_sdk=alipay-easysdk-net"));
            Assert.IsTrue(response.Body.Contains("sign"));
        }

        [Test]
        public void TestQuery()
        {
            AlipayUserCertifyOpenQueryResponse response = Factory.Member.Identification().Query("89ad1f1b8171d9741c3e5620fd77f9de");

            Assert.AreEqual(response.Code, "10000");
            Assert.AreEqual(response.Msg, "Success");
            Assert.IsNull(response.SubCode);
            Assert.IsNull(response.SubMsg);
            Assert.NotNull(response.HttpBody);
            Assert.AreEqual(response.Passed, "F");
            Assert.IsNull(response.IdentityInfo);
            Assert.AreEqual(response.MaterialInfo, "{}");
        }
    }
}

File: php/test/member/identification/ClientTest.php
<?php


namespace Alipay\EasySDK\Test\member\identification;


use Alipay\EasySDK\Kernel\Factory;
use Alipay\EasySDK\Member\Identification\Models\IdentityParam;
use Alipay\EasySDK\Member\Identification\Models\MerchantConfig;
use Alipay\EasySDK\Test\TestAccount;
use PHPUnit\Framework\TestCase;

class ClientTest extends TestCase
{
    protected $app = null;

    public function __construct($name = null, array $data = [], $dataName = '')
    {
        parent::__construct($name, $data, $dataName);
        $account = new TestAccount();
        $this->app = new Factory($account->getTestAccount());
    }

    public function testInit()
    {
        $identityParam = new IdentityParam();
        $identityParam->identityType = "CERT_INFO";
        $identityParam->certType = "IDENTITY_CARD";
        $identityParam->certName = "张三";
        $identityParam->certNo = "5139011988090987631";

        $merchantConfig = new MerchantConfig();
        $merchantConfig->returnUrl = "www.taobao.com";

        $result = $this->app->member()->identification()->init(microtime(),'FACE',$identityParam,$merchantConfig);
        $this->assertEquals('10000', $result['code']);
        $this->assertEquals('Success', $result['msg']);
    }

    public function testCertify()
    {
        $result = $this->app->member()->identification()->certify("16cbbf40de9829e337d51818a76eacc2");
        $this->assertEquals(true, strpos($result,'sign')>0);
        $this->assertEquals(true, strpos($result,'gateway.do')>0);
    }

    public function testQuery()
    {
        $result = $this->app->member()->identification()->query("16cbbf40de9829e337d51818a76eacc2");
        $this->assertEquals('10000', $result['code']);
        $this->assertEquals('Success', $result['msg']);

    }

}